### ISSUE TITLE
Make NAPALM more verbose

### ISSUE THREAD

**Issue by mirceaulinic** [napalm-automation/napalm-base#272]
Discussed privately, it would be nice to have at least a minimalist logging, in order to ease the debugging process.

**Comment by dbarrosop** [comment]
I agree, opened #273 as I think it's a dependency if we want to do it consistently.
